dapr/proto/runtime/v1/dapr.proto

```diff
@@ -29,6 +29,12 @@ service Dapr {
   // Invokes a method on a remote Dapr app.
   rpc InvokeService(InvokeServiceRequest) returns (common.v1.InvokeResponse) {}
 
+  // Check health of a component
+  rpc CheckHealth(CheckHealthRequest) returns (google.protobuf.Empty) {}
```

Comment:
This is a new endpoint, please mark it with an `Alpha1` suffix like our other alpha endpoints.